build.gradle

```diff
@@ -96,6 +96,12 @@ allprojects { subproj ->
 		mavenCentral()
 		maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
 	}
+	buildscript {
+		repositories {
+			// mavenLocal()
+			mavenCentral()
+		}
+	}
```

Comment:
Please move `mavenCentral()` to the `buildscript` / `repositories` block at the top of the file.